<launch>

  <node pkg="topic_tools" type="relay" args="/cmd_vel /mobile_base_controller/cmd_vel" name="cmd_vel_relayer">
  </node>

  <include file="$(find robotican_common)/launch/hector_slam.launch">
  </include>

  <include file="$(find hector_geotiff)/launch/geotiff_mapper.launch">
  </include>


  <arg name="record_interval" value="5"/>
  <arg name="bag_file_name" value="map.bag"/>
  <arg name="bag_file_dir" value="/home/lizi/maps"/>

  <node pkg="lizi_mapping" name="occupancy_grid_recorder" type="occupancy_grid_recorder.py" args="$(arg record_interval) $(arg bag_file_name) $(arg bag_file_dir)" output="screen">
  </node>

</launch>
